src/index.html

```diff
@@ -17,6 +17,92 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO Logo"
+        />
+      </a>
+      <nav class="navi-bar">
+        <ul class="navi-menu">
+          <li class="navi-item">
```

Comment:
The class name 'navi-item' could be simplified to 'nav__item' to follow the BEM naming convention.